lib/core/sidenav-layout/components/sidenav-layout/sidenav-layout.component.ts

```diff
@@ -81,14 +84,15 @@ export class SidenavLayoutComponent implements OnInit, AfterViewInit, OnDestroy
         this.mediaQueryList.removeListener(this.onMediaQueryChange);
     }
 
-    toggleMenu() {
+    toggleMenu(expandedSidenav) {
         if (!this.mediaQueryList.matches) {
             this.isMenuMinimized = !this.isMenuMinimized;
         } else {
             this.isMenuMinimized = false;
         }
 
         this.container.toggleMenu();
+        this.notify.emit((!this.isMenuMinimized).toString());
```

Comment:
Please emit a boolean. How we want to store this value should not have any effect on what is emitted.